samza-core/src/test/java/org/apache/samza/container/TestSamzaContainerExceptionHandler.java

```diff
@@ -17,20 +17,23 @@
  * under the License.
  */
 
-package org.apache.samza.container
+package org.apache.samza.container;
 
-import org.junit.Test
-import org.junit.Assert._
-import org.junit.Before
-import org.apache.samza.SamzaException
-import org.junit.After
+import org.apache.samza.SamzaException;
+import org.junit.Test;
```

Comment:
Makes sense, I thought the `junit` imports could have been potentially grouped together. I think I may have missed the `static` clause in the `assertTrue`.